plugin/src/test/resources/org/jenkinsci/plugins/workflow/cps/casc_test.yaml

```diff
@@ -0,0 +1,3 @@
+security:
+  cps:
+    hideSandbox: true
```

Comment:
Are we keeping this as a valid configuration? I think we should just release note this as a breaking change (which will likely affect nobody) and remove all the logic associated to `hideSandbox`  except what's needed for the data migration, and use the new APIs in script-security` directly when needed.

Reply:
Yes, if this casc script is used it will update the ScriptApproval one, but as we are removing the set method it will fail, so OK, I will remove the test related to CASC